extensions/npc/standalone.mjs

```diff
@@ -4,6 +4,16 @@
 import fs from 'fs';
 import InteractiveObjectServerBaseClass from '../../common/interactive-object/server.mjs';
 
+/**
+ * TODO: jsdoc
+ */
+class BulletinNPC extends InteractiveObjectServerBaseClass {
```

Comment:
I would recommend making this feature available to all interactive objects, and we'll not have to deal with different NPC class here.